azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java

```diff
@@ -1872,7 +1872,11 @@ private void processQueuedFlows(long activeExecutorsRefreshWindow,
           // process flow with current snapshot of activeExecutors
           selectExecutorAndDispatchFlow(reference, exflow, new HashSet<Executor>(activeExecutors));
         }
-        currentContinuousFlowProcessed++;
+
+        // do not count failed flow processsing (flows still in queue)
+        if(queuedFlows.getFlow(exflow.getExecutionId()) == null) {
+          currentContinuousFlowProcessed++;
```

Comment:
If a flow wakes up from sleep then its still in queue so we will not increment the count.